ocean_lib/data_provider/data_service_provider.py

```diff
@@ -382,7 +393,7 @@ def build_download_endpoint(provider_uri=None):
 
     @staticmethod
     def build_compute_endpoint(provider_uri=None):
-        return DataServiceProvider.build_endpoint('compute', provider_uri)
+        return DataServiceProvider.build_endpoint('computeStatus', provider_uri)
```

Comment:
This will not work, we have to replace this with specific compute endpoints. so one for computeStatus, computeStart, ...

Reply:
Ok, compute is now the computeStatus and I need to implement for the rest.